.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,45 @@
+name: Sonarcloud
+on: [push,pull_request]
+
+jobs:
+  call-build-workflow:
+    uses: ./.github/workflows/build.yml
+    secrets: inherit
```

Comment:
Does this allow you to call a workflow form inside a workflow? If so this is a game changing discovery!